plugin/storage/es/factory.go

```diff
@@ -186,6 +186,11 @@ func createSpanWriter(
 	return writer, nil
 }
 
+func createDependencyReader(client es.Client, logger *zap.Logger, cfg config.ClientBuilder) (dependencystore.Reader, error) {
```

Comment:
Thy introducing this method? It's oneliner, the line 190 can return directly.

Reply:
The intention was to maintain consistency with how SpanWriters/Readers were created as well, but agree it's an unnecessary change.